deploy/helm/values.yaml

```diff
@@ -216,6 +216,8 @@ trivy:
   #
   supportedConfigAuditKinds: "Workload,Service,Role,ClusterRole,NetworkPolicy,Ingress,LimitRange,ResourceQuota"
 
+  #subCommand. Either `image` or `filesystem` scanning. Depending on the target type required for the scan.
+  subCommand: image 
```

Comment:
@chen-keinan not sure if the PR is ready for review yet, though, why the helm config won't match the trivy-operator?